src/components/pageheader/pageheader.less

```diff
@@ -0,0 +1,4 @@
+.header {
+  font-family: Circular, sans-serif;
+  font-style: normal;
+}
```

Comment:
🤔 Font family should get set globally in Ant.D config, we might have to update it for this project if it's not correct.

Reply:
That might be part of the global styles ticket - I'm not sure. Should I take this out now and assume the config will get updated later?

Reply:
Yeah, it'll get updated in global styles, so you can take this out 👍